### PR TITLE
`EWMS_PILOT_QUARANTINE_TIME` + Pilot First-Message Timeout Fix [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
- Fixes bug with `SKYSCAN_MQ_CLIENT_TIMEOUT_WAIT_FOR_FIRST_MESSAGE` where it was ignored for rabbitmq
- Adds env var `EWMS_PILOT_QUARANTINE_TIME` that helps prevent blackhole condor nodes

### CODE REVIEW

**Review by tianluyuan — COMMENTED** [review]
Looks ok to me! I'm not entirely sure how it works so maybe nice to add some description, since there are multiple environment variables setting the timeouts.

File: resources/client_starter.py [hunk 1]
```diff
@@ -69,13 +69,20 @@ def make_condor_job_description(  # pylint: disable=too-many-arguments
     #   entrypoint, and loading the icetray env file
     #   directly from cvmfs messes up the paths" -DS
 
+    environment = ""
+    if not os.getenv("RABBITMQ_HEARTBEAT"):
+        environment = f"RABBITMQ_HEARTBEAT=600 {environment}"
+    if not os.getenv("EWMS_PILOT_QUARANTINE_TIME"):
+        environment = f"EWMS_PILOT_QUARANTINE_TIME=1800 {environment}"
```
[the remark this answers is not in the document]
Comment (ric-evans) [reply resources/client_starter.py:76]: https://github.com/icecube/skymap_scanner/blob/main/README.md#environment-variables-1

[the remark this answers is not in the document]
Comment (ric-evans) [reply resources/client_starter.py:76]: > # helps decrease condor blackhole nodes

[the remark this answers is not in the document]
Comment (ric-evans) [reply resources/client_starter.py:76]: But yeah, I agree there could be more documentation on this. For now, setting it to any value ~20mins should suffice

[the remark this answers is not in the document]
Comment (ric-evans) [reply resources/client_starter.py:76]: > any reason this is 1800 while in the README its 1200?

nope, setting it to any value ~20mins should suffice

**Pull request merged**